GAE/src/org/waterforpeople/mapping/app/web/SurveyedLocaleServlet.java

```diff
@@ -182,19 +179,16 @@ protected SurveyedLocaleResponse convertToResponse(List<SurveyedLocale> slList,
                                 // if the question has the allowOtherFlag set,
                                 // use OTHER as the device question type
                                 if (q.getAllowOtherFlag()) {
-                                    deviceQuestionType = "OTHER";
+                                    type = "OTHER";
                                 }
-                                questionTypeMap.put(sv.getSurveyQuestionId(),
-                                        deviceQuestionType);
+                                questionTypeMap.put(sv.getSurveyQuestionId(), type);
                             }
                         }
                     }
-                    if (!sv.getQuestionType().equals("IMAGE")) {
-                        // add question type
-                        siDto.addProperty(sv.getSurveyQuestionId(),
-                                sv.getStringValue() != null ? sv.getStringValue() : "",
-                                deviceQuestionType);
-                    }
```

Comment:
We always sync images, yes. The bit that kept me puzzled was that a `SurveyalValue`'s question type, as we saw the other day, is always *PHOTO*, not *IMAGE*, which is exactly the mapping happening a few lines above. So my concerns with that conditional check were:
* If question type is always *PHOTO*, the condition *always* holds true.
* Even if the question type is set to *IMAGE*, why would we cut it out from the responses?